Review.md

```diff
@@ -1,6 +1,13 @@
 # Please answer the following questions
 
 1.  In Jest, what are the differences between `describe()` and `it()` globals, and what are good uses for them?
+
+- Describe is a way to label a group of tests based on the larger concept they are testing. it() is another way to write test() and runs a specific test.
+
 2.  What is the point of `Test Driven Development`? What do you think about this approach?
-3.  What are `mocks`? What are a good use cases for them?
-4.  Mention three types of automated tests.
+
+- Test Driven Development helps ensure code is correct, it helps as a guide when refactoring, and helps avoid regressions - which is code that unexpectedly breaks. I like testing. It was hard to get used to at first, but I can see its benefits.
+
```

Comment:
Keep in mind that testing is one thing, Test Driven Development is the Red-Green-Refactor approach where tests are written and then code is written to get tests to pass